content/en/building/local-setup.md

```diff
@@ -219,7 +219,7 @@ git clone https://github.com/medic/cht-core.git
 - Run the following `cht-conf` command to compile and upload default test data to your local instance:
 
 ```shell
-cht --url=https://medic:password@localhost --accept-self-signed-certs csv-to-docs upload-docs
+cht --url=https://medic:password@localhost --accept-self-signed-certs
```

Comment:
We need to keep the `csv-to-docs upload-docs` too because the `cht` does not do it automatically.
```suggestion
cht --url=https://medic:password@localhost --accept-self-signed-certs
cht --url=https://medic:password@localhost --accept-self-signed-certs csv-to-docs upload-docs
```